eng/CredScanSuppression.json

```diff
@@ -90,7 +54,11 @@
         "sdk/identity/identity/test/azure-identity-chain-test.crt",
         "sdk/identity/identity/test/azure-identity-test.crt",
         "sdk/keyvault/keyvault-certificates/ca.key",
-        "sdk/identity/identity/assets/fake-cert.pem"
+        "sdk/identity/identity/assets/fake-cert.pem",
+        "sdk/storage/storage-blob/test/utils/constants.ts",
+        "sdk/servicebus/service-bus/test/public/testConstants.ts",
```

Comment:
We should name these files so that it is clear they are only containing fake test secrets so we don't accidently add other items in them that might cause a leak.

Reply:
```suggestion
        "sdk/servicebus/service-bus/test/public/fakeTestSecrets.ts",
```